src/Microsoft.Win32.Registry/src/Microsoft/Win32/Registry.cs

```diff
@@ -33,6 +33,10 @@ static class Registry
         /// <summary>Current Config Root Key. This is where current configuration information is stored.</summary>
         public static readonly RegistryKey CurrentConfig = RegistryKey.OpenBaseKey(RegistryHive.CurrentConfig, RegistryView.Default);
 
+#if MONO
+        public static readonly RegistryKey DynData = RegistryKey.OpenBaseKey(RegistryHive.DynData, RegistryView.Default);
```

Comment:
It needs Obsolete attribute but would be better just add it as partial class to Mono

Reply:
Done,